coollabsio/coolify
# Feat: New services part 3

## Note
- All templates with (tested and works) have been tested and fixed by myself and are now fully functional
- This PR does not only include PRs from contributors, I also added some templates myself.
- Templates form myself are: Freshrss Postgres, Freshrss MariaDB, Freshrss MySQL, Nextcloud MariaDB, Nextcloud MySQL

## Changes
- Fix: Nexctcloud
- Fix: Glitchtip
- Update Plausible
- Add CloudBeaver **(fixed + tested and works)**
- Add Freshrss sqlite **(fixed + tested and works)**
- Add Freshrss Postgres **(fixed + tested and works)**
- Add Freshrss MariaDB **(fixed + tested and works)**
- Add Freshrss MySQL **(fixed + tested and works)**
- Add Traccar **(fixed + tested and works)**
- Add Vvvweb **(fixed + tested and works)**
- Add Vvvweb MariaDB **(fixed + tested and works)**
- Add Vvvweb MySQL **(fixed + tested and works)**
- Add Nextcloud with Postgres **(fixed + tested and works)**
- Add Nextcloud with MariaDB **(fixed + tested and works)**
- Add Nextcloud with MySQL **(fixed + tested and works)**
- Add Kimai **(fixed + tested and works)**
- Add ntfy **(fixed + tested and works)**
- Add qbittorrent **(fixed + tested and works)**
- Add Homebox **(fixed + tested and works)**
- Add Owncloud **(fixed + tested and works)** -> I have disabled it for now as it also needs an FQDN (PR will fix this soon, it is the same problem as Azimut, you can leave it as it is for now, PR is in cooking in the Peak kitchen).
- Add Immich **(fixed + tested and works)**
- Add Cryptgeon **(fixed + tested and works)** -> The app itself throws and error when using but that is not our issue i think (@cupcakearmy see [comment](https://github.com/coollabsio/coolify/pull/3811#issuecomment-2402788320)).
- Add Affine **(fixed + tested and works)**
- Add flowise
- Add flowise with postgres
- Add Unsend
